### ISSUE TITLE
Implementing the calculation function (for Worker module - Turtlebot)

### ISSUE THREAD

**Issue by UmangBR** [amosproj/amos2022ss03-turtlebot-fleet-management#55]
## User story
1. As a SD
2. I want to implement the calculation function
3. So that I can calculate the direction and speed for the robot to reach the destination.

## Acceptance criteria
* Calculate the direction and speed for the robot to reach the destination.

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release
